.github/workflows/dev-build.yaml

```
@@ -7,61 +7,53 @@ on:
-          sudo apt-get install -y qemu-user-static
-
-      - name: Build Image
-        id: build
-        uses: redhat-actions/buildah-build@v2
+        uses: actions/checkout@v5
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to container registry
+        uses: docker/login-action@v2
```

actionlint (1.7.12)

[error] the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)